src/JuliaInterpreter.jl

```diff
@@ -114,6 +124,65 @@ end
 JuliaStackFrame(frame::JuliaStackFrame, pc::JuliaProgramCounter; kwargs...) =
     JuliaStackFrame(frame.code, frame, pc; kwargs...)
 
+"""
+`LocationInfo` contains information about the location of the source code for a `JuliaStackFrame`[@ref].
+
+It is constructed by calling `location(frame::JuliaStackFrame)`[@ref].
+
+Important fields:
+- `file`: Whether the method was defined in a file (`isfile == true`) or the REPL.
+- `filepath`: Path to the file of the method if it was defined in a file, otherwise `nothing`.
+- `data`: String for the method definition if it was defined in the REPL, otherwise `nothing`.
+- `line`: The current line of the expression about to be executed.
+- `defline`: The line where the method is defined.
+"""
+struct LocationInfo
+    infile::Bool
+    data::Union{Nothing, String}
+    filepath::Union{Nothing, String}
+    line::Int
+    defline::Int
+end
```

Comment:
Perhaps relevant: CodeTracking is ready to go live. I'm leaving [this PR](https://github.com/timholy/CodeTracking.jl/pull/2) open for a bit for comments, but once that merges I will register it.

Once that happens, I propose we use CodeTracking in this package. That will give us Revise integration, once I push a new version of Revise.

Reply:
Sure.